src/rfac/rfrdargs.c

```
@@ -0,0 +1,366 @@
+/*! \file
+ *  \brief File contains C implementation for \ref rfac_rdargs function.
+ */
+
+#include <math.h>
+#include <errno.h>
+#include <stdio.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#include <malloc.h>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)